postman/Get Clicks.postman_collection.json

```diff
@@ -0,0 +1,596 @@
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{base_url}}/accounts/{{accountId}}/lists",
+							"host": [
+								"{{base_url}}"
+							],
+							"path": [
+								"accounts",
+								"{{accountId}}",
+								"lists"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Random List",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "b41770d3-1044-48f5-9536-6cf9ba89088a",
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}",
+							"host": [
+								"{{base_url}}"
+							],
+							"path": [
+								"accounts",
+								"{{accountId}}",
+								"lists",
+								"{{listId}}"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Get Broadcast Open Clicks",
+			"item": [
+				{
+					"name": "Get Total Sent Broadcasts",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "e238f70c-3010-41a2-ba60-6fd4e312850d",
+								"exec": [
+									"pm.test(\"Sucessfully got total number of broadcasts\", function(){",
+									"    var jsonData = pm.response.json();",
+									"    if (pm.expect(jsonData.total_size).to.be.above(0)) {",
+									"        var broadcast_entry = Math.floor((Math.random() * jsonData.total_size ));",
+									"        pm.globals.set(\"broadcast_entry\", broadcast_entry );",
+									"    } else {",
+									"        pm.globals.set(\"broadcast_entry\",'' );",
+									"    }",
+									"    ",
+									"    ",
+									"})",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/broadcasts/total?status=sent",
+							"host": [
+								"{{base_url}}"
+							],
+							"path": [
+								"accounts",
+								"{{accountId}}",
+								"lists",
+								"{{listId}}",
+								"broadcasts",
+								"total"
+							],
+							"query": [
+								{
+									"key": "status",
+									"value": "sent"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Sent Broadcasts",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "b58a49c0-44d7-40fc-95df-f5057c64086a",
+								"exec": [
+									"var jsonData = pm.response.json();",
+									"pm.globals.set(\"broadcast_self_link\", '');",
+									"",
+									"pm.test('Successfully retrieved sent brroadcasts', function() {",
+									"    if (pm.expect(jsonData.entries.length).to.be.above(0)) {",
```

Comment:
You don't need to check the return value of this assertion. If it fails, the test will stop.